AlpsSDK/AlpsManager/AlpsManager.swift

```diff
@@ -10,7 +10,15 @@ import Foundation
 import CoreLocation
 import Alps
 
+public typealias OnMatchClojure = (_ matches: [Match], _ device: Device) -> Void
```

Comment:
you mean closure no ?

Reply:
yeah :D good catch.

Mistake made because of https://clojure.org/